packages/nuxt/src/runtime/composables.ts

```diff
@@ -0,0 +1,6 @@
+import { useNuxtApp } from '#imports'
+import { defineStore } from 'pinia'
+
+export const usePinia = () => useNuxtApp().$pinia
```

Comment:
do we really need this? This sohuld be the same as `getActivePinia()`

Reply:
Hmm then i guess not if we already have a composable to access it. Does it return same reference of `$pinia`? (then probably plugin injection is not necessary too). 

One small benefit of nuxt composables vs vue composables is that they also work outside of vue setup lifecycle with better accessibility support.